### ISSUE TITLE
Best Practices For Training Mode

### ISSUE THREAD

**Issue by kyleLesack** [timothyjamesbecker/FusorSV#19]
Hello,

I am using SVE/FusorSV for calling structural variants in _C. elegans_. My plan is to use mock genomes with simulated structural variants as training data to generate a model to use with my real sequencing data. Because the _C. elegans_  genome is only 100Mb, I have created multiple mock genomes for each variant type and for different size ranges. For example, I created individual mock genomes with:

30 deletions between 100bp and 1Kbp 
30 deletions between 1Kbp and 10Kbp
30 deletions between 10Kbp and 100Kbp

I also have done the same for duplications and inversions. I also used a sequencing simulator to generate reads at the same sequencing depth as the read sequencing data that I am using. I also have vcf files with the truth for each mock genome that I am using as the S0.vcf files to create the model.

I would appreciate some feedback on this strategy. Is it okay to have a only single type of structural variant in each mock genome? Are the size ranges reasonable? Do you have any other comments or suggestions?

Thanks in advance!

**Comment by timothyjamesbecker** [comment]
Hello, I am not sure if I have already replied and if not, sorry for the
slow response.  Your plan sounds reasonable, in that as long as you have a
good starting reference (or you are constructing your own) and using the
same callers from the paper, you will get somewhat similar results,
especially if you are making your own new training model.  The difference
could be large depending on the coverage and other factors, so I do
recommend
 training with your own S0.vcf truth files which may require that you
simulate more than you wished at the beginning.  As far are structural
variation types, you should mix them to ensure you are creating a plausible
simulation that will reflex some of what will be observed in the real
species data.  Please fee free to follow up.

all the best,

-Tim

**Comment by kyleLesack** [comment]
Thanks for the response. I ended up choosing my variant sizes based on the bin sizes in the "bin_map.json" file. The _C. elegans_ genome is of very high quality, so that isn't a concern for me. I also made sure that the depth of coverage in my simulated data matches that for the real sequencing data that I am working with. 

Having a docker image available for SVE was very helpful for variant calling. I was able to get results from all of the callers except for GenomeSTRiP and Breakseq2. I think these require human data, so they probably aren't candidates anyways. 

It does make sense to combine the different variant types in each S0.vcf file, so I'll try that. Feel free to close this issue unless you have any other suggestions.

Thanks!
